.github/workflows/ci.yml

```diff
@@ -1,186 +1,236 @@
 ---
 name: "CI"
-on:  # yamllint disable-line rule:truthy
-  - "push"
-  - "pull_request"
+concurrency:  # Cancel any existing runs of this workflow for this same PR
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+on: # yamllint disable-line rule:truthy rule:comments
+  push:
+    branches:
+      - "main"
```

Comment:
Wouldn't this cause CI to stop running on ltm PRs after this is merged

Reply:
Isn't that what the following handles?
```
  pull_request: ~
```

Reply:
Ah, I think this would prevent releases from the ltm-1.6 branch, updating.

Reply:
That handles the checks in the PR but after it's merged into main, ltm or develop we run additional checks.

Reply:
This looks good now though